src/main/java/core/basesyntax/dao/impl/OperationDaoImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.OperationDao;
+import core.basesyntax.db.StorageImpl;
+
+public class OperationDaoImpl implements OperationDao {
+    @Override
+    public void put(String key, Integer value) {
+        StorageImpl.fruits.put(key, value);
```

Comment:
Use putIfAbsent instead of put to handle null values and avoid potential issues.